tests/providers/test_bedrock_mantle.py

```
@@ -0,0 +1,313 @@
+        bedrock_client=bedrock_client,
+        mantle_openai_client=invalid_openai_client,
+        mantle_anthropic_client=anthropic_client,
+    )
+    unconfigured_provider = BedrockProvider(bedrock_client=bedrock_client)
+
+    assert openai_provider.mantle_base_url == 'https://openai.example.com'
+    assert anthropic_provider.mantle_base_url == 'https://anthropic.example.com'
+    assert standard_openai_provider.mantle_base_url == 'https://standard-openai.example.com'
+    assert fallback_provider.mantle_base_url == 'https://anthropic.example.com'
+    with pytest.raises(UserError, match='pass `mantle_base_url`'):
+        unconfigured_provider.mantle_base_url
```

Ruff (0.15.21)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)